torch/ao/quantization/_dbr/model_utils.py

```diff
@@ -102,10 +102,13 @@ def attach_scale_zp_values_to_model(
     if hasattr(module, '_auto_quant_state'):
         qstate: AutoQuantizationState = module._auto_quant_state  # type: ignore[assignment]
         for tensor_id, observer in qstate.tensor_id_to_observer.items():
-            scale, zp = observer.calculate_qparams()
-            # tensor_id_to_observer is a ModuleDict which has to have string keys
-            # tensor_id_to_scale_zp is a normal dict which can have int keys
-            qstate.tensor_id_to_scale_zp[int(tensor_id)] = (scale, zp)
+            activation_int8_quantized = \
```

Comment:
I think we also have some utility functions for this (may need some cleanup): https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/utils.py#L98

Reply:
yeah, existing function takes qconfig, not sure if it's worth creating a new util function with only one callsite for now

Reply:
I see, sounds good, is there a way to keep track of this so that we don't miss it in the future when we expand this condition?

Reply:
maybe we can think about it when there is a second callsite? Not sure if it's worth to do this now when we don't when/if the second callsite would appear.

Reply:
(motivation: https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)

Reply:
sure, sounds good